test/Controllers/UsersTest.php

```diff
@@ -69,87 +68,96 @@ public function testCreateUser() {
             $response->user->permalink,
             $response->user->timezone
         );
-        // print_r($responseBody);
+    }
 
-        $this->assertEquals($createUserBody, $responseBody);
 
-        // Can't delete created user ATM.
+    /**
+     * Test POST /users/new
+     * GonebusyLib\Controllers\UsersController::createUser()
+     */
+    public function testCreateUser() {
+        $createUserBody = $this->uniqueUserData();
+
+        // Create GonebusyLib\Models\EntitiesUserResponse:
+        $response = $this->users->createUser(Configuration::$authorization, $createUserBody);
+
+        // Was it created?
+        $this->assertObjectHasAttribute('user', $response);
+
+        // Does it have all the original data we sent?
+        $responseBody = $this->dataFromResponse($response);
+        $this->assertEquals($responseBody, $createUserBody);
+
+        // Can't delete test users at the moment.
     }
 
     /**
      * Test GET /users/{id}
      * GonebusyLib\Controllers\UsersController::getUserById()
      */
     public function testGetUserById() {
-        // $id = 'id';
-        // $response = $this->users->getUserById(
-        //     $this->authorization,
-        //     $id);
-        // // print_r($response);
-        //
-        // $this->assertObjectHasAttribute('user', $response);
-        // $this->assertEquals($id, $response->user->accountManagerId);
+        // Create user:
+        $createUserBody = $this->uniqueUserData();
+        $createResponse = $this->users->createUser(Configuration::$authorization, $createUserBody);
+
+        // Get user by it's id
+        $response = $this->users->getUserById(Configuration::$authorization, $createResponse->user->id);
+
+        // Was it fetched?
+        $this->assertObjectHasAttribute('user', $response);
+
+        // Does it have all the original data we sent?
+        $responseBody = $this->dataFromResponse($response);
+        $this->assertEquals($responseBody, $createUserBody);
     }
 
     /**
      * Test PUT /users/{id}
      * GonebusyLib\Controllers\UsersController::updateUserById()
      */
     public function testUpdateUserById() {
-        // // $this->expectException(EntitiesErrorException::class);
-        //
-        // $id = 'id';
-        // $updateUserByIdBody = new UpdateUserByIdBody();
-        // $response = $this->users->updateUserById(
-        //         $this->authorization,
-        //         $id,
-        //         $updateUserByIdBody);
-        // print_r($response);
-
-        // $this->assertObjectHasAttribute('user', $response);
-        // $this->assertEquals($id, $response->user->accountManagerId);
+        // Create user:
+        $createUserBody = $this->uniqueUserData();
+        $createResponse = $this->users->createUser(Configuration::$authorization, $createUserBody);
+
+        $anotherUserBody = $this->uniqueUserData();
+        # XXX ^ Uses CreateUserBody doubling as UpdateUserByIdBody.
+
```

Comment:
`$anotherUserBody` is actually a `CreateUserBody` model instance, NOT a `UpdateUserByIdBody`. `updateUserById()` still accepts it though, so I let it be. Should I use `UpdateUserByIdBody` for the sake of correctness though?

p.s. notice `CreateUserBody` and `UpdateUserByIdBody` are almost the same but for some reason APImattic changed the order in which it processes their method arguments...
See [v0.0.8/src/Models/CreateUserBody.php#L64-L85](https://github.com/gonebusy/gonebusy-php-client/blob/v0.0.8/src/Models/CreateUserBody.php#L64-L85) vs. [v0.0.8/src/Models/UpdateUserByIdBody.php#L63-L84](https://github.com/gonebusy/gonebusy-php-client/blob/v0.0.8/src/Models/UpdateUserByIdBody.php#L63-L84) .